services/blockstorage/service.go

```diff
@@ -112,6 +126,7 @@ func (s *service) updateLastCommittedBlock(block *protocol.BlockPairContainer) {
 	s.lastBlockLock.Lock()
 	defer s.lastBlockLock.Unlock()
 
+	s.metrics.blockHeight.Inc()
```

Comment:
I'd do something different, since as soon as we're persisting block heights, this Gauge will lose sync with the block storage. Instead, add an `Update` method on the `Gauge` type and read the block height from `block`

Reply:
Done.